README.md

```diff
@@ -129,11 +129,13 @@ Create hot-plug virtio-blk devices via API Server:
 
 ```
 sudo ./dbs-cli  \
-  --api-sock-path [socket path]
+  --api-sock-path [socket path] update
```

Comment:
For update subcommand, is it necessary to have a `hotplug` prefix? It is a quite burden on typing it from my perspective.